src/core/log.c

```diff
@@ -70,7 +70,7 @@ void mLog(int category, enum mLogLevel level, const char* format, ...) {
 	va_start(args, format);
 	if (context) {
 		if (!context->filter || mLogFilterTest(context->filter, category, level)) {
-			context->log(context, category, level, format, args);
+			(context->log)(context, category, level, format, args);
```

Comment:
I don't understand why this is necessary. I do x->y(x, ...) all over the place in mGBA; why is it a problem here?

Reply:
It seems to be conflicting with something
src/core/log.c:73:55: error: macro "log" passed 5 arguments, but takes just 1

Reply:
`log` isn't a macro in mGBA code. Just undef it.

Reply:
The previous defination is in the ppu maths library.. which doesn't sound like a good idea to undef.

Reply:
this is a C file, not a header. it's safe to undef because nothing past the includes could possibly use it

Reply:
done